library_generation/model/generation_config.py

```diff
@@ -25,7 +25,7 @@
 COMMON_PROTOS_LIBRARY_NAME = "common-protos"
 GAPIC_GENERATOR_VERSION = "gapic_generator_version"
 LIBRARIES_BOM_VERSION = "libraries_bom_version"
-GENERATOR_VERSION_ENV_KEY = "GENERATOR_VERSION"
+GENERATOR_VERSION_ENV_KEY = "DOCKER_GAPIC_GENERATOR_VERSION"
```

Comment:
Now that we have a `DOCKER_GAPIC_GENERATOR_VERSION` specified in the docker file, I think we can remove the usage of `GENERATOR_VERSION` now.

Reply:
I thought I removed all of the references to the GENERATOR_VERSION env var but there was a missing one in `.github/scripts/hermetic_library_generation`. I also renamed the variable in line 26 to make it clear that it's a reference to the config yaml key

Reply:
Thanks, this is much clearer now! As I mentioned in another comment, maybe we can try to get rid of `GAPIC_GENERATOR_VERSION_CONFIG_KEY` all together.

Reply:
Got rid of it